src/pages/policy.jsx

```diff
@@ -1,10 +1,99 @@
 import React from "react";
 import PageLayout from "../common/layout/PageLayout";
+import { H1, H4, P } from "../common/components/elements/Text";
 
 export default function Policy() {
   return (
-    <PageLayout>
-      <div>Policy page...</div>
+    <PageLayout className="text-primary-900 items-stretch">
```

Comment:
remove mt-5 from all and give gap-5 in PageLayout
```suggestion
    <PageLayout className="text-primary-900 items-stretch gap-5">
```